cmd/dockerd/daemon_unix.go

```diff
@@ -4,21 +4,18 @@ package main
 
 import (
 	"context"
-	"net"
-	"os"
-	"os/signal"
-	"path/filepath"
-	"strconv"
-	"time"
```

Comment:
Check your IDE settings, these shouldn't be moved.

Reply:
@olljanat this has been rectified. Thank you.